features/your_contributions_section.feature

```diff
@@ -0,0 +1,25 @@
+Feature:
```

Comment:
Another good information. I based the name on the same for the step 1 feature (your_details_section.feature). So let's continue with this discussion but your spot/suggestion (which is good) could be handle in a different PR to treat two feature files.